include/zephyr/bluetooth/gatt.h

```diff
@@ -55,14 +55,18 @@ enum {
 	/** @brief Attribute read permission with authentication.
 	 *
 	 *  If set, requires encryption using authenticated link-key for read
-	 *  access.
+	 *  access. This further means that if set BT_GATT_PERM_READ_ENCRYPT is
+	 *  implied, and can be omitted. It does however not imply
+	 *  BT_GATT_PERM_READ_128B_ENTROPY.
```

Comment:
It was a suggestion from me: It may not be clear to everyone not familiar with GATT that `AUTHEN` implies `ENCRYPT`, but since we haven't seen many (any?) issues with it, I guess we can omit it again.

Reply:
Since 128B is a bit special, a detailed description is nice I think. But it might be sufficient that we have it under the `128B_ENTROPY` then?